.travis.yml

```diff
@@ -24,20 +24,28 @@ env:
 
 jobs:
   include:
-    - name: "Test the Frontend"
-      install:
-        - "bin/installDeps.sh"
-        - "export GIT_HASH=$(git rev-parse --verify --short HEAD)"
-        - "npm install ep_test_line_attrib"
-      script:
-        - "tests/frontend/travis/runner.sh"
-    - name: "Test the Dockerfile"
+    - name: "Run the Backend"
       install:
         - "cd src && npm install && cd -"
       script:
         - "docker build -t etherpad:test ."
         - "docker run -d -p 9001:9001 etherpad:test && sleep 3"
         - "cd src && npm run test-container"
+        - "npm run test"
```

Comment:
why don' we put this in a dedicated section, so we have separated tests for Docker build (& its minimal testing), and the backend testing itself?

Reply:
could do, I'm just playing with something that should be stable for now.